css/colors.scss

```diff
@@ -1,2 +1,4 @@
 $white: #FFFFFF;
-$primary-color: #2f1c6a;
\ No newline at end of file
+$primary-color: #2f1c6a;
+$secound-color: #673de6;
```

Comment:
Pisze sie `second`, a nie `secound`. Ogolnie to chyba powinno byc `secondary`, bo wyzej jest `primary`